src/pages/issues.tsx

```diff
@@ -9,36 +8,88 @@ import {
     NO_ISSUES_FOUND_MESSAGE,
 } from '@/components/constants/messages';
 import { ISSUES_URL } from '@/components/constants/url';
+import axios from 'axios';
+
+type SearchFieldProps = {
+    searchText: string;
+    onSearchTextChanged: (event: any) => void;
+    onSearchTextSubmitted: () => void;
+    loading: boolean;
+};
+
+const SearchField = ({
+    searchText,
+    onSearchTextChanged,
+    onSearchTextSubmitted,
+    loading,
+}: SearchFieldProps) => {
+    return (
+        <div className={classNames.searchFieldContainer}>
+            <input
+                placeholder="Enter query string to search issues"
+                value={searchText}
+                onChange={onSearchTextChanged}
+                className={classNames.issueSearchInput}
+            />
+            <button
+                onClick={onSearchTextSubmitted}
+                className={classNames.issuesSearchSubmitButton}
+                disabled={loading || !searchText.trim()}
+            >
+                Submit
+            </button>
+        </div>
+    );
+};
 
 const Issues: FC = () => {
     const [issueList, setIssueList] = useState<[]>([]);
+    const [searchText, setSearchText] = useState('');
+    const [isLoading, setIsLoading] = useState(true);
+    const [error, setError] = useState<null | any>(null);
 
-    const { response, error, isLoading } = useFetch(ISSUES_URL, {
-        data: {
-            userData: {
-                roles: {
-                    super_user: true,
-                },
-            },
-        },
-    });
+    const fetchIssues = async () => {
+        try {
+            setIsLoading(true);
+            const res = await fetch(`${ISSUES_URL}?q=${searchText}`);
```

Comment:
Why are we writing fetch and not using RTK?

Reply:
As already mentioned in the previous PRs, the issues to task conversion v1 feature needs to be live ASAP in prod. Hence, it was discussed to pick up RTK query and mutation for future updates.

Reply:
But it will take additional effort - thus pushing the review and hence merge further. 
There are dependencies in the backend that are in pause for this PR. 

I'm not saying no to the implementation, but it was decided to release the initial version ASAP and worry about enhancements later as this is an important feature.

Reply:
Here are the RFC tickets to pick up RTK query as an enhancement for one-click issue to task conversion: 

https://github.com/Real-Dev-Squad/todo-action-items/issues/149
https://github.com/Real-Dev-Squad/todo-action-items/issues/141